plugin_gems.rb

```diff
@@ -27,3 +27,11 @@
 download "fluent-plugin-systemd", "1.0.2"
 download "opencensus", "0.5.0"
 download "opencensus-stackdriver", "0.3.0"
+if windows?
+  download 'windows-pr', '1.2.6'
```

Comment:
Are single quotes significant? If not, can they be consistent with the rest of the file?

Reply:
It works.